sdk/core/Azure.Core/src/Shared/HashCodeCombiner.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Shared/HashCodeBuilder.cs